packages/semantic-ui/src/components/List.js

```diff
@@ -533,6 +582,29 @@ const useList = (WrappedComponent: ComponentType<any>) => (
       );
     }
 
+    /**
+   * Renders the CSV export button.
+   *
+   * @returns {null|*}
+   */
+    renderCsvExportButton() {
+      if (!this.props.csvExportButton) {
+        return null;
+      }
+
+      return (
+        <Button
+          basic
+          color={this.props.csvExportButton.color}
+          key={BUTTON_KEY_CSV_EXPORT}
+          onClick={this.onCsvExportButton.bind(this)}
+        >
+          <Icon name='shopping basket' />
```

Comment:
Is there significance to the "shopping basket" icon? Would a "list" or "cloud download" icon be more appropriate?